src/datagen/java/dev/compactmods/machines/datagen/lang/EnglishLangGenerator.java

```diff
@@ -120,6 +121,7 @@ protected void addTranslations() {
         add("jei.compactmachines.machines", "Machines are used to make pocket dimensions. Craft a machine and place it in world, then use a Personal Shrinking Device to go inside.");
         add("jei.compactmachines.shrinking_device", "Use the Personal Shrinking Device (PSD) on a machine in order to enter a compact space. " +
                 "You can also right click it in the overworld for more info.");
+        add("death.attack." + MachineVoidAir.DAMAGE_SOURCE.msgId, "%1$s Died from Machine Void Air");
```

Comment:
Would prefer to align with other death messages, ie "`player` failed to enter the void"